control-plane/p4RuntimeSerializer.cpp

```diff
@@ -578,11 +596,8 @@ class P4RuntimeSymbolTable {
         function(symbols);
 
         // Now that the symbol table is initialized, we can compute ids.
-        symbols.computeIdsForSymbols(P4RuntimeSymbolType::ACTION);
-        symbols.computeIdsForSymbols(P4RuntimeSymbolType::TABLE);
-        symbols.computeIdsForSymbols(P4RuntimeSymbolType::ACTION_PROFILE);
-        symbols.computeIdsForSymbols(P4RuntimeSymbolType::COUNTER);
-        symbols.computeIdsForSymbols(P4RuntimeSymbolType::METER);
+        for (auto& table : symbols.symbolTables)
+            symbols.computeIdsForSymbols(table.first);
```

Comment:
You'll have a conflict with @antoninbas's PR: https://github.com/p4lang/p4c/pull/717 on this.
That said, this is making the implementation more scalable when we add other symbol types.